internal-webapps/org.wso2.bfsi.authentication.endpoint/src/main/webapp/privacy_policy.jsp

```diff
@@ -0,0 +1,238 @@
+                    <p>The organization running WSO2 Open Banking may revise the Privacy Policy from time to time. You can find the most recent governing policy with the respective link provided by the organization running WSO2 IS 5.5. The organization will notify any changes to the privacy policy over our oficial public channels.</p>
+                </section>
+                
+                <section style="margin-bottom: 2em;">
+                    <h3 id="your-choices" style="font-weight:400; color:black">Your choices</h3>
+                    <p>If you are already have a user account within WSO2 Open Banking, you have the right to deactivate your account if you find that this privacy policy is unacceptable to you.</p>
+                    <p>If you do not have an account and you do not agree with our privacy policy, you can chose not to create one.</p>
+                </section>
+                
+                <section style="margin-bottom: 2em;">
+                    <h3 id="contact-us" style="font-weight:400; color:black">Contact us</h3>
+                    <p>Please contact WSO2 if you have any question or concerns regarding this privacy policy.</p>
+                    <p><a href="https://wso2.com/contact/">https://wso2.com/contact/</a></p>
+                </section>
+                
+                <section style="margin-bottom: 2em;">
+                    <h2 id="disclaimer" style="font-weight:400; color:black">Disclaimer</h2>
+                    <ol>
+                        <li>WSO2, its employees, partners, and affiliates do not have access to and do not require, store, process or control any of the data, including personal data contained in WSO2 Open Banking. All data, including personal data is controlled and processed by the entity or individual running WSO2 Open Banking.  WSO2, its employees partners and affiliates are not a data processor or a data controller within the meaning of any data privacy regulations.  WSO2 does not provide any warranties or undertake any responsibility or liability in connection with the lawfulness or the manner and purposes for which WSO2 Open Banking is used by such entities or persons.<br><br></li>
+                        <li>This privacy policy is for the informational purposes of the entity or persons  running WOS2 IS 5.5.0 and sets out the processes and functionality contained within WSO2 Open Banking regarding personal data protection. It is the responsibility of entities and persons running WSO2 Open Banking to create and administer its own rules and processes governing users’ personal data, and such  rules and processes may change the use, storage and disclosure policies contained herein. Therefore users should consult the entity or persons running WSO2 Open Banking for its own privacy policy for details governing users’ personal data. </li>
```

Comment:
```suggestion
                        <li>This privacy policy is for the informational purposes of the entity or persons  running WOS2 IS 7.0 and sets out the processes and functionality contained within WSO2 Open Banking regarding personal data protection. It is the responsibility of entities and persons running WSO2 Open Banking to create and administer its own rules and processes governing users’ personal data, and such  rules and processes may change the use, storage and disclosure policies contained herein. Therefore users should consult the entity or persons running WSO2 Open Banking for its own privacy policy for details governing users’ personal data. </li>
```

Reply:
Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/8/commits/55101612f0aa99105f83a6d7d3d648000eb13112